hack/local-up-cluster.sh

```diff
@@ -37,6 +37,10 @@ FAIL_SWAP_ON=${FAIL_SWAP_ON:-"false"}
 # Name of the dns addon, eg: "kube-dns" or "coredns"
 DNS_ADDON=${DNS_ADDON:-"coredns"}
 CLUSTER_CIDR=${CLUSTER_CIDR:-10.1.0.0/16}
+ALLOCATE_NODE_CIDRS=${ALLOCATE_NODE_CIDRS:-false}
+RANGE_ALLOCATOR=${RANGE_ALLOCATOR:-"RangeAllocator"}
+DUALSTACK_ENABLED=${DUALSTACK_ENABLED:-"false"}
```

Comment:
this probably should mean "everything dualstack", ie. service cidrs too `SERVICE_CLUSTER_IP_RANGE`

Reply:
https://github.com/kubernetes/kubernetes/pull/109090/commits/2379e60aa66c43bf193680d650a15858570d6608#r939158788